test_unstructured_inference/models/test_donut.py

```diff
@@ -0,0 +1,69 @@
+import pytest
+import unstructured_inference.models.donut as donut
+
+from transformers import DonutSwinModel
+from PIL import Image
+
+
+@pytest.mark.parametrize(
+    "model_path, processor_path, config_path",
+    [
+        ("crispy_donut_path", "crispy_proc", "crispy_config"),
+        ("cherry_donut_path", "cherry_proc", "cherry_config"),
```

Comment:
No sprinkle donuts?

Reply:
sprinkle_donut name is reserved for the best model version 😁 I promise. We are still learning to bake them perfectly 🤤